src/ai/backend/logging/types.py

```diff
@@ -0,0 +1,15 @@
+import enum
+
+
+class LogLevel(enum.StrEnum):
+    CRITICAL = "CRITICAL"
+    ERROR = "ERROR"
+    WARNING = "WARNING"
+    INFO = "INFO"
+    DEBUG = "DEBUG"
+    NOTSET = "NOTSET"
+
+
+class LogFormat(enum.StrEnum):
+    SIMPLE = enum.auto()
+    VERBOSE = enum.auto()
```

Comment:
When defining `StrEnum`, there are cases where some variants are defined as uppercase strings (`LogLevel`) and others use `enum.auto()` (`LogFormat`)

Unless there's a specific reason for the difference, wouldn't it be better to standardize on one method?

Reply:
The stdlib doc says `enum.auto()` in `StrEnum` defaults to the lowercased name of the field. This is fine for most cases while `LogLevel` is _an exception_ to use the uppercased names as the values.

Reply:
I'll test if the lowercased values work fine in `LogLevel`.

Reply:
I've added `CIStrEnum` and `CIUpperStrEnum` classes to handle this case, as the stdlib enum offers `_generate_next_value_()` method that can be overriden.